telegram/telegram.go

```diff
@@ -21,7 +21,6 @@ func responseHandler(target string, message string, sender *bot.User) {
 		return
 	}
 	msg := tgbotapi.NewMessage(id, message)
-	msg.ReplyToMessageID = msg.ReplyToMessageID
```

Comment:
Why did you kill this?

Reply:
The linter hinted that it was assigned to itself, I don't see the point of this code.

Reply:
Oh totally. Kill it with fire.